flake8_django/checkers/model_meta.py

```diff
@@ -0,0 +1,97 @@
+import ast
+
+from flake8_django.checkers.base_model_checker import BaseModelChecker
+from flake8_django.checkers.issue import Issue
+
+
+class DJ09(Issue):
+    code = 'DJ09'
+    description = 'Model: {model} must define class Meta'
+
+
+class DJ10(Issue):
+    code = 'DJ10'
+    description = 'Model: {model} has no verbose_name'
+
+
+class DJ11(Issue):
+    code = 'DJ11'
+    description = 'Model: {model} has no verbose_name_plural'
+
+
+class ModelMetaChecker(BaseModelChecker):
+    model_name_lookup = 'Model'
+
+    def checker_applies(self, node):
+        for base in node.bases:
+            if self.is_model_name_lookup(base) or self.is_models_name_lookup_attribute(base):
+                return True
```

Comment:
@avallbona could you add a check here for abstract model, if it's an abstract model we shouldn't be throwing an error. There is `is_abstract_model` on `BaseModelChecker` that you can use for that purpose.

Reply:
done